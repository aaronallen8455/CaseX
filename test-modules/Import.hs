module Import
  ( Foo(..)
  ) where

data Foo = Foo | Bar | Baz
