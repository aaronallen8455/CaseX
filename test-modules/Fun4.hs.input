import Import

main :: IO ()
main = pure ()

test :: Foo -> Bool
test foo =
  let f :: Foo -> Bool
      f SPLIT = True
  in f foo
