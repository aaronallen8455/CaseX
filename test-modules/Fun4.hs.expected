import Import

main :: IO ()
main = pure ()

test :: Foo -> Bool
test foo =
  let f :: Foo -> Bool
      f Foo = True
      f Bar = True
      f Baz = True
  in f foo
